### PR TITLE
Added 'Package' section in contribution documentation

### PR DESCRIPTION
**Pull request by grayjeremy**

### CODE REVIEW

**Comment by damccorm** [comment]
I don't think we should take this. https://docs.microsoft.com/en-us/azure/devops/extend/develop/add-build-task?view=azure-devops&viewFallbackFrom=vsts contains these instructions, and we link to that in the README.

**Comment by grayjeremy** [comment]
Creating a custom build task and updating the azure-pipelines-tasks are _significantly_ different.  It took me a long time to tease out the differences, hence the PR.

**Comment by damccorm** [comment]
Oh, I see - misunderstood the original intent of this PR. I think `package` as a title might be a little misleading, really what we're after here is `integration testing` or something along those lines. We are packaging as part of that, but its not really the purpose, right?

I think this feels like a good addition, I'll add some comments

File: docs/contribute.md [hunk 1]
```diff
@@ -124,3 +124,32 @@ For a specific task
 ```bash
 node make.js testLegacy --task Xcode
 ```
+
+# Package
```
Comment (damccorm) [inline docs/contribute.md]: Maybe `# Integration Testing`?

File: docs/contribute.md [hunk 2]
```diff
@@ -124,3 +124,32 @@ For a specific task
...
+
+After a local code update and testing, you should test any changes in your Azure DevOps organization before submitting a pull request. 
+
+## Prepare your organization
+In order to upload an extension directly to your Azure DevOps organization you must be an [administrator](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/pools-queues?view=vsts#security) of "All agent pools" or have a PAT token associated with an administrator.
```
Comment (damccorm) [inline docs/contribute.md]: Maybe add a recommendation along the lines of "we recommend using a dedicated organization for tests like this"

File: docs/contribute.md [hunk 3]
```diff
@@ -124,3 +124,32 @@ For a specific task
...
 ```
+
+# Package
+
+After a local code update and testing, you should test any changes in your Azure DevOps organization before submitting a pull request. 
+
+## Prepare your organization
+In order to upload an extension directly to your Azure DevOps organization you must be an [administrator](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/pools-queues?view=vsts#security) of "All agent pools" or have a PAT token associated with an administrator.  
+
+## Update metadata (optional)
```
Comment (damccorm) [inline docs/contribute.md]: I'd remove the optional tag here - if people don't do this we risk weird conflicts, and even though its technically optional I don't really want to call attention to that. I usually read optional as "skip this step unless you have a reason not to"

**Pull request closed without merging**